src/main/scala/com.snowplowanalytics/forex/Forex.scala

```diff
@@ -35,7 +34,7 @@ trait CreateForex[F[_]] {
 object CreateForex {
   def apply[F[_]](implicit ev: CreateForex[F]): CreateForex[F] = ev
 
-  implicit def syncCreateForex[F[_]: Sync: ZonedClock](implicit
+  implicit def syncCreateForex[F[_]: Monad: Transport](implicit
```

Comment:
Nice, so it seems we don't need `Sync` here 👌 Does it mean we could also remove `id` instance we have below?

Reply:
I think so, removing `Id` instance